include/deal.II/distributed/shared_tria.h

```diff
@@ -106,6 +113,39 @@ namespace parallel
                                          const std::vector< CellData< dim > > &cells,
                                          const SubCellData &subcelldata);
 
+      /**
+       * Return auxilary vector subdomain_id_with_artificial .
```

Comment:
typo: auxiliary. same in the next function.

Since this is a public function, it would be good to have a longer description what the vector it returns represents -- I have no idea either from this function nor from the documentation of the vector it probably returns. (This might also suggest that you're really computing/returning something that is so specialized that it would better be implemented close to the algorithm that actually needs it, rather than in a public interface. But I may be misunderstand what the function does.)